packages/edge-login-ui-rn/src/reducers/WorkflowReducer.ts

```diff
@@ -1,62 +1,115 @@
-import { WorkflowName, workflows } from '../constants/workflows'
+import { SceneNames } from '../constants/Scenes'
 import { Action } from '../types/ReduxTypes'
 
-export interface WorkflowState {
-  readonly currentKey: WorkflowName
-  readonly currentSceneIndex: number
+export interface SceneState {
+  readonly currentScene: SceneNames
 }
 
-const initialState: WorkflowState = {
-  currentKey: 'loadingWF',
-  currentSceneIndex: 0
+const initialState: SceneState = {
+  currentScene: SceneNames.loadingScene
 }
 
-export function workflow(
-  state: WorkflowState = initialState,
+// TODO: remove back/next, update to specific scene
+export function scene(
```

Comment:
Since this is now the scene reducer, I think we should rename the file to match. I really like the new name, BTW.